ceph.spec.in

```diff
@@ -1894,6 +1894,7 @@ fi
 %{_datadir}/ceph/mgr/test_orchestrator
 %{_datadir}/ceph/mgr/volumes
 %{_datadir}/ceph/mgr/zabbix
+%{_datadir}/ceph/mgr/rgw
```

Comment:
it looks like the other modules in this list are sorted in alphabetical order. Maybe move this between `restful` and `selftest`?

Reply:
fixed